pages/apim/3.x/installation-guide/configuration/reporters/installation-guide-reporters.adoc

```diff
@@ -0,0 +1,201 @@
+| Description
+| Default value
+
+| `enabled`
+| Weither to start the file reporter or not
+| false
+
+| `fileName`
+| File name to write access logs (use '%s-yyyy_mm_dd' pattern to create one file per event type on daily base)
+| *#{systemProperties['gravitee.home']}/metrics/%s-yyyy_mm_dd}*
+
+| `output`
+| Type of output file (json, message_pack, elasticsearch, csv)
+| *json*
+
+| `flushInterval`
+| interval between file flush (in ms)
+| *1000*
+
+| `retainDays`
+| The number of days to retain files before deleting one
+| *0 (to retain forever)*
+
+| `<EVENT_TYPE>.exclude`
+| Fields to exclude from the output (available for json` and `message_pack` outputs only)
+| *none*
+
+| `<EVENT_TYPE>.include`
+| Fields to include in the output (available for `json` and `message_pack` outputs and only if excludes have been defined)
+| *none*
+
+| `<EVENT_TYPE>.rename`
+| Fields to rename when writing the output (available for `json`` and `message_pack` outputs only)
+| *none*
+|===
+
+NOTE: <EVENT_TYPE> refers to the kind of event reported by the gateway and can be either `request`, `log`, `node` or `healthcheck`
+
+==== Example
+
+Here is a configuration example that will exclude all fields from the request JSON file except the `api` and
+`application` fields, renaming the `application` field to `app`, and will not report `log`, `node` or `healthcheck` events
+
+[source,yaml]
+----
+reporters:
+  file:
+    enabled: true 
+    fileName: ${gravitee.home}/metrics/%s-yyyy_mm_dd
+    output: json 
+    request: 
+      exclude: 
+        - "*"
+      include: 
+        - api
+        - application
+      rename: 
+        application: app
+    log:
+      exclude: *
+    node:
+      exclude: *
+    healthcheck:
+      exclude: *
+----
+
+NOTE: the fileName property **must** end with the `%s-yyyy_mm_dd` pattern, otherwise your file will not be written.
+
+=== TCP Reporter
+
+==== Configuration Parameters
+[cols="1,1,1"]
+|===
+| Parameter name
+| Description
+| Default value
+
+| `enabled`
+| Weither to start the TCP reporter or not
+| false
+
+| `output`
+| Format of the data written to the TCP socket (json, message_pack, elasticsearch, csv)
+| *json*
+
+| `host`
+| The TCP host where the event should be published (can be a valid host name or IP address)
+| *localhost*
+
+| `port`
+| The TCP port used to connect to the host
+| *8123*
+
+| `connectTimeout`
+| Maximum time allowed to establish the TCP connection in milliseconds
+| *10000*
+
+| `reconnectAttempts`
+| How many time the socket should try to establish a connection in case of failure
+| *10*
+
+| `reconnectInterval`
+| Time in millisceconds between each socket connection attempt
+| *500*
+
+| `retryTimeout`
+| If the max reconnect attempts have been reached, how many time in milliseconds the reporter should wait before trying to connect again
+| *5000*
+
+|===
+
+==== Example
+
+We can define the same configuration as above, but write the events to a TCP socket instead of a file using
```

Comment:
```suggestion
The following example uses the same configuration as the previous example above, however it writes the events to a TCP socket instead of a file.
```